language-server/modules/langserver-core/src/test/resources/completion/listener_decl/config/config8.json

```diff
@@ -8,7 +8,7 @@
     {
       "label": "test",
       "kind": "Variable",
-      "detail": "none",
+      "detail": "$UndefinedType$",
```

Comment:
Shall we create an issue under LS label to address `$UndefinedType$` being set as the detail. A minor change is needed at `AbstractCompletionProvider.java:193`.